agent/directory.go

```diff
@@ -56,7 +56,14 @@ func (a *Agent) removeTaskDirectory(tc *taskContext) {
 	}
 	grip.Infof("Deleting directory for completed task: %s", tc.taskDirectory)
 
-	err := os.RemoveAll(tc.taskDirectory)
+	// Removing long relative paths hangs on Windows https://github.com/golang/go/issues/36375,
+	// so we have to convert to an absolute path before removing.
+	wd, err := os.Getwd()
```

Comment:
https://golang.org/pkg/path/filepath/#Abs is much less scary than os.Getwd

Reply:
fixed